cogboard-app/src/main/kotlin/com/cognifide/cogboard/storage/model/Board.kt

```diff
@@ -0,0 +1,11 @@
+package com.cognifide.cogboard.storage.model
+
+data class Board(
```

Comment:
Represent whole configuration as an object, Board seems to be only part of the config.